cmd/config.go

```diff
@@ -164,7 +164,6 @@ type Config struct {
 	PA PAConfig
 
 	Common struct {
-		BaseURL string
```

Comment:
So I think you need to restore this field and the line in `cmd/boulder-wfe/main.go` that sets `wfe.BaseURL = c.Common.BaseURL`. Otherwise we won't get the nice safe switchover we wanted.